boot.php

```diff
@@ -782,7 +781,7 @@ function run_update_function($x)
 			//send the administrator an e-mail
 			DBStructure::updateFail(
 				$x,
-				sprintf(t('Update %s failed. See error logs.'), $x)
+				sprintf(L10n::t('Update %s failed. See error logs.'), $x)
```

Comment:
Can you please replace all the `sprintf(t('...'), ...$args)` constructs with `L10n::t('...',  ...$args)`?